src/regional_settings.h

```diff
@@ -182,54 +182,90 @@ struct overmap_feature_flag_settings {
     overmap_feature_flag_settings() = default;
 };
 
-struct overmap_forest_settings {
+om_settings_forest_id const bogus_forest_id;
```

Comment:
Because this is not initialized with a value, the id is `""`. Same for all the other bogus ids.